feconf.py

```diff
@@ -28,22 +28,19 @@
 # code in core/platform.
 PLATFORM = 'gae'
 
-# This should be string comparison, since all environment variables
-# are converted to string
-IS_MINIFIED = os.environ.get('MINIFICATION') == 'True'
-
 # Whether we should serve the development or production experience.
 # DEV_MODE should only be changed to False in the production environment.
-# To use minified resources in the development environment,
-# change the MINIFICATION env variable in app.yaml to True.
+# Or if you want to use minified resources in the development environment,
+# by changing the DEV_MODE env variable in app.yaml to False.
```

Comment:
Whether we should serve the development or production experience. DEV_MODE should only be changed to False in the production environment, or if you want to use minified resources in the dev env.

Reply:
done